contributors/guide/README.md

```diff
@@ -55,11 +55,12 @@ Please make sure to read and observe our [Code of Conduct](https://github.com/cn
 
 If you haven’t set up your environment, please find resources [here](/contributors/devel).
 
-## Community Expectations
+## Community Expectations and Roles
 
 Kubernetes is a community project. Consequently, it is wholly dependent on its community to provide a productive, friendly and collaborative environment.
 
-Read and review the [Community Expectations](community-expectations.md) for an understand of code and review expectations. 
+- Read and review the [Community Expectations](community-expectations.md) for an understand of code and review expectations. 
+- See [Community Membership](/community-membership.md) for a list the various responsibilities of contributor roles. 
```

Comment:
Could we add some new contributor context for this? New contributors aren't members, after all.

Maybe: "We encourage regular contributors to grow into community membership roles; please see a list of the various responsibilities here".